README.md

```diff
@@ -321,6 +321,24 @@ with:
   set-distribution-checksum: false
 ```
 
+---
+
+### `release-channel`
+
+| Name | Description | Required | Default |
+| --- | --- | --- | --- |
+| `release-channel` | Which release channel to use: `stable` or `release-candidate`. | No | `stable` |
```

Comment:
`nightly` would be another option for this but this use case is probably even rarer than the release candidate.